feature-libs/asm/asm-customer-360/sections/asm-customer-product-reviews/asm-customer-product-reviews.component.spec.ts

```diff
@@ -96,7 +103,7 @@ describe('AsmCustomerProductReviewsComponent', () => {
         rating: '3.5',
         reviewStatus: 'Complete',
         reviewText: '',
-        item: 'Shirt, SKU: shirt',
+        item: 'Shirt, asm.customer360.productReviews.sku shirt',
```

Comment:
We probably want the colon outside the translation string. 
```suggestion
        item: 'Shirt, asm.customer360.productReviews.sku: shirt',
```